server/routes/liteCategories.js

```diff
@@ -313,24 +317,44 @@ module.exports = function Index({ formService, offendersService, userService, au
           data: { details },
         })
       } else {
-        await offendersService.approveLiteCategorisation({
-          context: res.locals,
-          bookingId: bookingIdInt,
-          sequence: assessmentData.sequence,
-          approvedDate,
-          supervisorCategory,
-          approvedCategoryComment,
-          approvedCommittee,
-          nextReviewDate,
-          approvedPlacement,
-          approvedPlacementComment,
-          approvedComment,
-          transactionalClient: transactionalDbClient,
-        })
-        res.redirect(`/liteCategories/confirmed/${bookingId}`)
+        try {
+          await offendersService.approveLiteCategorisation({
+            context: res.locals,
+            bookingId: bookingIdInt,
+            sequence: assessmentData.sequence,
+            approvedDate,
+            supervisorCategory,
+            approvedCategoryComment,
+            approvedCommittee,
+            nextReviewDate,
+            approvedPlacement,
+            approvedPlacementComment,
+            approvedComment,
+            transactionalClient: transactionalDbClient,
+          })
+          res.redirect(`/liteCategories/confirmed/${bookingId}`)
+        } catch (error) {
+          if (assessmentHasBeenApprovedInNomisMannually(error)) {
+            formService.deleteLiteCategorisation(bookingId, assessmentData.sequence, transactionalDbClient)
+            res.redirect(`/liteCategories/alreadyApproved/${bookingId}`)
+          } else {
+            throw error
+          }
+        }
       }
     })
   )
 
+  router.get(
+    '/alreadyApproved/:bookingId',
+    asyncMiddleware(async (req, res) => {
+      const { bookingId } = req.params
+      const user = await userService.getUser(res.locals)
+      res.locals.user = { ...user, ...res.locals.user }
```

Comment:
The endpoint getCategoryHistory() uses would return assessment details from nomis but sadly the approver is not included - it would need to be added in prison-api. probably not worth it atm